src/main/kotlin/gg/flyte/christmas/util/Extn.kt

```diff
@@ -50,13 +51,13 @@ fun String.colourise(): String {
 
         val builder = StringBuilder()
         replaceSharp.forEach { c ->
-            builder.append("&").append(c)
+            builder.append("§").append(c)
         }
 
         message = message.replace(hexCode, builder.toString())
     }
 
-    return ChatColor.translateAlternateColorCodes('&', message)
+    return ChatColor.translateAlternateColorCodes('§', message)
```

Comment:
While arguably both the section sign and the ampersand symbols are legacy and outdated, various components in the plugin use ampersands rather than section signs for menu work. Ampersands are also notably easier to type out. 

I would recommend reverting this.

Reply:
Alright, simple change